docs/src/web-client/create_deploy_tutorial.md

```diff
@@ -55,27 +55,43 @@ pnpm install
 Install the Miden WebClient SDK:
 
 ```bash
-pnpm i @demox-labs/miden-sdk@0.6.1-next.4
+pnpm i @demox-labs/miden-sdk@0.8.1
```

Comment:
I think here we already have 0.8.2

https://www.npmjs.com/package/@demox-labs/miden-sdk?activeTab=readme

I think we can use that with React and Vue.js now. Does it make sense to change the tutorial?